universal-application-tool-0.0.1/test/models/ApplicantTest.java

```diff
@@ -45,4 +46,22 @@ public void createsOnlyOneApplicantData() {
 
     assertThat(applicant.getApplicantData()).isEqualTo(applicantData);
   }
+
+  @Test
+  public void mergeApplicantData() {
+    ApplicantData data1 = new Applicant().getApplicantData();
+    Path foo = Path.create("$.applicant.foo");
+    Path subMapFoo = Path.create("$.applicant.subObject.foo");
+    Path subMapBar = Path.create("$.applicant.subObject.bar");
+    data1.putString(foo, "foo");
+    data1.putString(subMapFoo, "also_foo");
+    ApplicantData data2 = new Applicant().getApplicantData();
+    data2.putString(foo, "bar");
+    data2.putString(subMapBar, "bar");
+    data1.putString(subMapFoo, "also_foo");
+    List<Path> removedPaths = data1.mergeFrom(data2);
```

Comment:
(blocking) please add spaces between test setup, the `merge` method, and the asserts

Reply:
Done.